src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -592,7 +592,15 @@ define('diagramStore', ['jquery', 'xwiki-meta', 'draw.io', 'xwiki-events-bridge'
       <cache>long</cache>
     </property>
     <property>
-      <code>define('diagramEditor', ['jquery', 'diagramStore'], function($, diagramStore) {
+      <code>define('diagramEditor', ['jquery', 'diagramStore', 'xwiki-meta'], function($, diagramStore, xm) {
```

Comment:
``xwiki-meta`` (``xm``) is not needed anymore AFAICS, since the code to upload the image was moved to ``diagramStore``.